src/classes/BDI_DataImportService.cls

```diff
@@ -38,9 +38,19 @@
 *
 */
 global with sharing class BDI_DataImportService {
+    private static DescribeFieldResult batchNumberDescribe = UTIL_Describe.getFieldDescribe(
+        UTIL_Namespace.StrTokenNSPrefix('DataImport__c'),
+        UTIL_Namespace.StrTokenNSPrefix('NPSP_Data_Import_Batch__c')
+    );
```

Comment:
This is better - using the UTIL_Describe class - but this block should still avoid introducing new string references to objects/fields this way:

```
    private static DescribeFieldResult batchNumberDescribe = UTIL_Describe.getFieldDescribe(
            String.valueOf(DataImport__c.SObjectType),
            String.valueOf(DataImport__c.NPSP_Data_Import_Batch__c)
    );

```

Reply:
agreed. I updated this to use String.valueOf() on the SObject/SObjectField tokens instead

Reply:
👍🏽